common/changes/@itwin/core-backend/mike-imodeldb-error_2025-05-21-17-35.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Updates iModelDb Error Handling",
```

Comment:
```suggestion
      "comment": "",
```

Reply:
you should definitely mention this in the changelog